### ISSUE TITLE
arc_summary3 always installed, RPM not installable on build system

### ISSUE THREAD

**Issue by DeHackEd** [zfsonlinux/zfs#7651]
### System information

Type | Version/Name
 --- | --- 
Distribution Name	| CentOS
Distribution Version	| 6.x
Linux Kernel	| n/a, userspace only
Architecture	| x86_64
ZFS Version	| zfs-0.7.0-1614-g<a href="https://github.com/zfsonlinux/zfs/commit/7637ef8">7637ef8</a> (current git head at time of writing)
SPL Version	| n/a, userspace only

### Describe the problem you're observing
```
$ configure --with-config=user
...
$ make rpm-utils
...
Wrote: /tmp/zfs-build-dhe-Y4fH1Ok6/RPMS/x86_64/zfs-0.7.0-1446_g7637ef8.el6.x86_64.rpm
...
$ rpm -qp --requires zfs-0.7.0-1446_g7637ef8.el6.x86_64.rpm
...
/usr/bin/python3  
...
```

There is not, and never was, a python3 available on the system. But the RPM requires it and can't be installed without it, or at least without `--nodeps` (obviously not desired)

`arc_summary3.py` is installed onto the system regardless of what is available on the local operating system. Though this is independent of the RPM dependency, this ends up producing an RPM with impossible, and unnecessary, dependencies (we already have the old version of the script).

### Describe how to reproduce the problem
See above

### Include any warning/errors/backtraces from the system logs
N/A

**Comment by behlendorf** [comment]
It sounds like we should add a configure option to optionally disable anything which requires python 3.

**Comment by DeHackEd** [comment]
Appears to have been fixed by #8096, closing.
